recipes/r-gifti/meta.yaml

```diff
@@ -0,0 +1,78 @@
+{% set version = '0.8.0' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-gifti
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/gifti_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/gifti/gifti_{{ version }}.tar.gz
+  sha256: 9eea71aa93358d4560cd66d39d1f66bc8d5ac920eb3f979af46a274b33e963bd
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  noarch: generic
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{ posix }}zip               # [win]
+    - cross-r-base {{ r_base }}    # [build_platform != target_platform]
+  host:
+    - r-base
+    - r-r.utils
+    - r-base64enc
+    - r-xml2 >=1.1.1
+  run:
+    - r-base
+    - r-r.utils
+    - r-base64enc
+    - r-xml2 >=1.1.1
+
+test:
+  commands:
+    - $R -e "library('gifti')"           # [not win]
+    - "\"%R%\" -e \"library('gifti')\""  # [win]
+
+about:
+  home: https://CRAN.R-project.org/package=gifti
+  license: GPL-2.0-or-later
```

Comment:
```suggestion
  license: GPL-2.0-only
```

Reply:
Thanks for pointing this out. It has been fixed in the latest commit